<?xml version="1.0" encoding="utf-8"?>
<LOG>
  <LOG ID="1540043809" Data="01/06/2016 19:56:21">
    <Sessao>ivekoik0rkwizrgeyrvvtegb</Sessao>
    <IP>::1</IP>
    <HostName>DESKTOP-SLG6G19\andre</HostName>
    <Mensagem>Erro de rede ou específico à instância ao estabelecer conexão com o SQL Server. O servidor não foi encontrado ou não estava acessível. Verifique se o nome da instância está correto e se o SQL Server está configurado para permitir conexões remotas. (provider: SQL Network Interfaces, error: 26 - Erro ao Localizar Servidor/Instância Especificada) *** The underlying provider failed on Open.</Mensagem>
    <NameSpace>App_Dominio.Security.EmpresaSecurity`1[[App_Dominio.Entidades.App_DominioContext, App_Dominio, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]] =&gt; home/default</NameSpace>
    <Context />
    <StackTrace>   em System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   em System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   em System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.&lt;&gt;c__DisplayClass7.&lt;GetResults&gt;b__5()
   em System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   em System.Data.Entity.Core.Objects.ObjectQuery`1.&lt;System.Collections.Generic.IEnumerable&lt;T&gt;.GetEnumerator&gt;b__0()
   em System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   em System.Linq.Enumerable.SingleOrDefault[TSource](IEnumerable`1 source)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.&lt;GetElementFunction&gt;b__2[TResult](IEnumerable`1 sequence)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.ExecuteSingle[TResult](IEnumerable`1 query, Expression queryRoot)
   em System.Data.Entity.Core.Objects.ELinq.ObjectQueryProvider.System.Linq.IQueryProvider.Execute[TResult](Expression expression)
   em System.Linq.Queryable.SingleOrDefault[TSource](IQueryable`1 source)
   em System.Data.Entity.Internal.Linq.InternalSet`1.FindInStore(WrappedEntityKey key, String keyValuesParamName)
   em System.Data.Entity.Internal.Linq.InternalSet`1.Find(Object[] keyValues)
   em System.Data.Entity.DbSet`1.Find(Object[] keyValues)
   em App_Dominio.Security.EmpresaSecurity`1._ValidarSessao(String sessionId) na C:\sistemas\app_dominio\App_Dominio\App_Dominio\Security\EmpresaSecurity.cs:linha 202</StackTrace>
  </LOG>
  <LOG ID="199480260" Data="01/06/2016 19:59:38">
    <Sessao>ivekoik0rkwizrgeyrvvtegb</Sessao>
    <IP>::1</IP>
    <HostName>DESKTOP-SLG6G19\andre</HostName>
    <Mensagem>The underlying provider failed on Open.</Mensagem>
    <NameSpace>App_Dominio.Security.EmpresaSecurity`1[[App_Dominio.Entidades.App_DominioContext, App_Dominio, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]] =&gt; </NameSpace>
    <Context>__RequestVerificationToken="lsKx3V6YYld8kTaCK5McZHORnK0P5sOGmZlgTATvQ3hqaSAfCzhlleekbcLqPS0TsuJm3VCyW38W4k0ER-KWdQ23qOG4q9mE9HDijReXkt81",UserName="borgeslealandre@gmail.com",Password="evergladesMega"</Context>
    <StackTrace />
  </LOG>
  <LOG ID="1389164225" Data="01/06/2016 19:59:38">
    <Sessao>ivekoik0rkwizrgeyrvvtegb</Sessao>
    <IP>::1</IP>
    <HostName>DESKTOP-SLG6G19\andre</HostName>
    <Mensagem>The underlying provider failed on Open.</Mensagem>
    <NameSpace>App_Dominio.Security.EmpresaSecurity`1[[App_Dominio.Entidades.App_DominioContext, App_Dominio, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null]] =&gt; </NameSpace>
    <Context>__RequestVerificationToken="lsKx3V6YYld8kTaCK5McZHORnK0P5sOGmZlgTATvQ3hqaSAfCzhlleekbcLqPS0TsuJm3VCyW38W4k0ER-KWdQ23qOG4q9mE9HDijReXkt81",UserName="borgeslealandre@gmail.com",Password="evergladesMega"</Context>
    <StackTrace />
  </LOG>
</LOG>